app/views/layouts/application.html.erb

```diff
@@ -4,7 +4,7 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="msvalidate.01" content="3667D2FEB238C1BF174F3CB1AE0D3C16" />
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true %>
```

Comment:
hmm now looking at it think they can be left in they shouldn't be doing any harm, that being said though something is causing extra 'pageview' events from being fired. The platform page produced some consistently for me